src/main/scala/com/github/xiaodongw/swagger/finatra/SwaggerController.scala

```diff
@@ -6,17 +6,21 @@ import com.twitter.finatra.response.Mustache
 import io.swagger.models.Swagger
 import io.swagger.util.Json
 
-@Mustache("index")
-case class SwaggerView(title: String, path: String)
-
 class SwaggerController(docPath: String = "/api-docs", swagger: Swagger) extends Controller {
+
   get(s"${docPath}/model") { request: Request =>
     response.ok.body(Json.mapper.writeValueAsString(swagger))
       .contentType("application/json").toFuture
   }
 
   get(s"${docPath}/ui") { request: Request =>
-    response.temporaryRedirect
-        .location("/webjars/swagger-ui/2.2.8/index.html?url=/api-docs/model")
+    redirectToSwaggerUi()
   }
+
+  get(s"${docPath}") { request: Request =>
+    redirectToSwaggerUi()
+  }
+
+  private def redirectToSwaggerUi() =
+    response.temporaryRedirect.location(s"/webjars/swagger-ui/2.2.8/index.html?url=${docPath}/model")
```

Comment:
I'd make this a relative path, so that if this application is hosted in something like an AWS ALB with path based routing that it still works on the subpath